server/datastore/mysql/migrations/data/20210817112308_MigrateAppConfig.go

```diff
@@ -0,0 +1,109 @@
+package data
```

Comment:
I'm not sure that this should be run in a `data` migration rather than in a regular `table` migration. I think this could be a problem when we remove the old table, as `data` migrations will run _after all_ `table` migrations -- the table would be dropped before the data would be migrated.

I'm really struggling to identify there being a good purpose for the existence of `data` migrations, and this is an example of where it is causing pain.